src/utils/push.rs

```diff
@@ -91,6 +98,16 @@ pub async fn push_profile(
         a => return Err(PushProfileError::BuildExitError(a)),
     };
 
+    if ! Path::new(format!("{}/deploy-rs-activate", deploy_data.profile.profile_settings.path).as_str()).exists() {
+        return Err(PushProfileError::DeployRsActivateDoesntExist);
+    }
+
+    if ! Path::new(format!("{}/activate-rs", deploy_data.profile.profile_settings.path).as_str()).exists() {
```

Comment:
This is good, but can we add this to one of our built-in flake checks too (like we do with `deploy-rs-activate`?)

Reply:
Ah, yes, forgot about it.